Lib/test/test_operator.py

```diff
@@ -328,6 +328,14 @@ def test_bitwise_xor(self):
         self.assertRaises(TypeError, operator.xor, None, None)
         self.assertEqual(operator.xor(0xb, 0xc), 0x7)
 
+    def test_is_none(self):
+        operator = self.module
+        a = 'xyzpdq'
```

Comment:
I know that this is not really required, but I would prefer to have at least one "falsy" but not-`None` value to be tested :)

Reply:
Three lines below, there is `self.assertFalse(operator.is_none(a))`. Maybe I misunderstood your comment, though 😅